app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -0,0 +1,118 @@
+angular.module('containerDashboard', ['ui.bootstrap', 'patternfly', 'patternfly.charts'])
+    .config(['$httpProvider', function($httpProvider) {
+        $httpProvider.defaults.headers.common['X-CSRF-Token'] = jQuery('meta[name=csrf-token]').attr('content');
+    }])
+    .controller('containerDashboardController', ['$scope', '$http', '$interval', "$location",
+        function($scope, $http, $interval, $location) {
+            document.getElementById("center_div").className += " miq-body";
```

Comment:
Do you really need it there? Why is this #center_div different from every other?